### PR TITLE
Fix Python 3.7 support.

### PR DESCRIPTION
**Pull request by stevenbronson-wk**
#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: runtime.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# https://github.com/dependabot/dependabot-core/blob/8ab4d78efe7cf9a75bef76dd883d7ee3fffffb40/python/lib/dependabot/python/file_parser/python_requirement_parser.rb#L76-L85
+python-version=3.7.13
```
Comment (austinmatherne-wk) [inline runtime.txt:2]: 3.7.14?

**Review by austinmatherne-wk — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -10,8 +10,8 @@ Cheroot==8.6.0
 pycountry==22.3.5
 # plugins
 # EdgarRenderer plugin
-NumPy==1.23.3
-Matplotlib==3.6.0
+NumPy==1.21.6
```
Comment (austinmatherne-wk) [inline requirements.txt:13]: The version range in pyproject.toml should also be updated

Comment (austinmatherne-wk) [reply requirements.txt:13]: Discussed offline. pip should handle installing a version from the range in pyproject.toml that's compatible with the current python runtime

**Review by derekgengenbacher-wf — APPROVED** [review]

**Pull request merged**